policyengine_us/parameters/gov/states/household/state_ctcs.yaml

```diff
@@ -0,0 +1,28 @@
+description: All state Child Tax Credits.
+values:
+  0000-01-01:
+    - az_dependent_tax_credit  # Arizona Dependent Tax Credit
+    - ca_yctc  # California Young Child Tax Credit
+    - co_ctc  # Colorado Child Tax Credit
```

Comment:
The CO CTC was adopted in 2022 - I am not sure if the value will be returned as 0 or n/a when testing for 2021

Reply:
similar concern for all periods before 2021 - could we add on integration test for all of the created variables just to see if it is supported for each year (e.g. 2020 - 2024)